frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.tsx

```diff
@@ -432,22 +448,29 @@ export const WhatsNewMenu = (props: Props) => {
   const phoneAnnouncement: WhatsNewEntry = {
     title: l10n.getString("whatsnew-feature-phone-header"),
     snippet: l10n.getString("whatsnew-feature-phone-snippet"),
-    content: (
-      <WhatsNewContent
-        description={l10n.getString("whatsnew-feature-phone-description")}
-        heading={l10n.getString("whatsnew-feature-phone-header")}
-        image={PhoneMaskingHero.src}
-        videos={{
-          // Unfortunately video files cannot currently be imported, so make
-          // sure these files are present in /public. See
-          // https://github.com/vercel/next.js/issues/35248
-          "video/webm; codecs='vp9'":
-            "/animations/whatsnew/phone-masking-hero.webm",
-          "video/mp4": "/animations/whatsnew/phone-masking-hero.mp4",
-        }}
-        cta={ctaUpgradePhones}
-      />
-    ),
+    content:
+      props.runtimeData && isPhonesAvailableInCountry(props.runtimeData) ? (
+        <WhatsNewContent
+          description={l10n.getString("whatsnew-feature-phone-description")}
+          heading={l10n.getString("whatsnew-feature-phone-header")}
+          image={PhoneMaskingHero.src}
+          videos={{
+            // Unfortunately video files cannot currently be imported, so make
+            // sure these files are present in /public. See
+            // https://github.com/vercel/next.js/issues/35248
+            "video/webm; codecs='vp9'":
+              "/animations/whatsnew/phone-masking-hero.webm",
+            "video/mp4": "/animations/whatsnew/phone-masking-hero.mp4",
+          }}
+          cta={
+            <CtaLinkButton
+              subscribed={props.profile.has_phone}
+              link={getPhoneSubscribeLink(props.runtimeData, "monthly")}
+              label={l10n.getString("whatsnew-feature-phone-upgrade-cta")}
```

Comment:
`<CtaLinkButton>` determines whether to show an upgrade CTA to a user, by checking if they already possess the plan in question. It can also be reused in future What's New entries with a upgrade CTA, like the previous Phone masking and Bundle announcements.